docs/manual/external-tools.tex

```diff
@@ -750,6 +753,21 @@
 \url{https://youtrack.jetbrains.com/issue/IDEA-159286}.
 
 
+\subsectionAndLabel{Running a checker on every IntelliJ change or save}{intellij-every-save}
+
+To make IntelliJ compile on every change or save,
+access Settings (Preferences on macOS).
+Select Build, Execution, Deployment > Compiler.
```

Comment:
I prefer linking to the IntelliJ help page instead of writing text here.  The help page detects if you are using a Mac and shows the correct commends.  Plus it gives more information and stays up-to-date.

Reply:
I think this is the link for this setting:

https://www.jetbrains.com/help/idea/compiling-applications.html#run_packaged_jar